### DISCUSSION TITLE
Convective Outlooks

### DISCUSSION THREAD

**Discussion by coolkid0281** [niels747/2D-Weather-Sandbox#74] [Ideas]
I was wondering if it is possible to make a Convective Outlook for the game or not? Like, just add like a little box somewhere where it is easy to see. I understand if you can't do days out like 2–3 days out but if you could add like 1 day outlook and when you change like  Land Evaporation, Lake / Sea Evaporation, Evaporation Heat, Lake / Sea Temp. I was thinking that you could add an outlook Like in one of these red boxes 
![image](https://user-images.githubusercontent.com/110744575/226192398-e0e251c2-6586-4e9e-898b-1e59a96c4f49.png)

**Comment by wilburcoding** [comment]
umm
not sure how you implement this on a 2d sim
